File: TM 15 C#-Basics/C#-Basics/Strings/program2.cs
// Tek Raj Joshi
// Superset ID: 1368453

using System;
public class Program2
{
    public static void Main()
    {
        string str;
        int l = 0;
        Console.Write("Input the string : ");
        str = Console.ReadLine();
        while (l < str.Length)
        {
            Console.Write("{0} ", str[l]);
            l++;
        }
        Console.Write("\n\n");
    }
}

File: TM 5 IO Operations/IO Operations/program2.py
# Tek Raj Joshi
# Superset ID: 1368453

no_of_lines_to_display = int(input("Enter the number of lines you want to display: "))
with open('filename.txt', mode='r', encoding = 'utf-8') as f:
    for line in range(no_of_lines_to_display):
        print(f.readline())
    f.close()